2024-07-29 08:00:43.035 [info] Extension host with pid 1165 started
2024-07-29 08:00:43.065 [info] Lock '/home/shahnawaz_nazimali/.codeoss/data/User/workspaceStorage/35b348cb/vscode.lock': Lock acquired.
2024-07-29 08:00:45.179 [info] ExtensionService#_doActivateExtension ms-python.python, startup: false, activationEvent: 'onLanguage:python'
2024-07-29 08:00:45.180 [info] ExtensionService#_doActivateExtension vscode.emmet, startup: false, activationEvent: 'onLanguage'
2024-07-29 08:00:46.182 [info] ExtensionService#_doActivateExtension vscode.git-base, startup: true, activationEvent: '*', root cause: vscode.git
2024-07-29 08:00:46.530 [info] ExtensionService#_doActivateExtension vscode.git, startup: true, activationEvent: '*'
2024-07-29 08:00:46.530 [info] ExtensionService#_doActivateExtension vscode.github, startup: true, activationEvent: '*'
2024-07-29 08:00:47.888 [info] Eager extensions activated
2024-07-29 08:00:47.888 [info] ExtensionService#_doActivateExtension EditorConfig.EditorConfig, startup: false, activationEvent: 'onStartupFinished'
2024-07-29 08:00:47.889 [info] ExtensionService#_doActivateExtension google.cloud-shell, startup: false, activationEvent: 'onStartupFinished'
2024-07-29 08:00:47.890 [info] ExtensionService#_doActivateExtension googlecloudtools.cloudcode, startup: false, activationEvent: 'onStartupFinished'
2024-07-29 08:00:47.890 [info] ExtensionService#_doActivateExtension vscode.debug-auto-launch, startup: false, activationEvent: 'onStartupFinished'
2024-07-29 08:00:47.891 [info] ExtensionService#_doActivateExtension vscode.merge-conflict, startup: false, activationEvent: 'onStartupFinished'
2024-07-29 08:00:51.767 [info] ExtensionService#_doActivateExtension vscode.json-language-features, startup: false, activationEvent: 'onLanguage:jsonc'
2024-07-29 08:00:51.768 [info] ExtensionService#_doActivateExtension vscode.typescript-language-features, startup: false, activationEvent: 'onLanguage:jsonc'
2024-07-29 08:41:20.371 [error] [ms-python.python] provider FAILED
2024-07-29 08:41:20.421 [error] Error: AssertionError: Cannot infer the keyword <Keyword: from>
    at ae (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2306827)
    at oe (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2304787)
    at Immediate.<anonymous> (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2301081)
    at processImmediate (node:internal/timers:476:21)
2024-07-29 08:47:19.617 [error] [ms-python.python] provider FAILED
2024-07-29 08:47:19.617 [error] Error: UnboundLocalError: local variable 'start_index' referenced before assignment
    at ae (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2306827)
    at oe (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2304787)
    at Immediate.<anonymous> (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2301081)
    at processImmediate (node:internal/timers:476:21)
2024-07-29 08:47:29.110 [error] [ms-python.python] provider FAILED
2024-07-29 08:47:29.110 [error] Error: UnboundLocalError: local variable 'start_index' referenced before assignment
    at ae (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2306827)
    at oe (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2304787)
    at Immediate.<anonymous> (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2301081)
    at processImmediate (node:internal/timers:476:21)
2024-07-29 08:47:45.400 [error] [ms-python.python] provider FAILED
2024-07-29 08:47:45.401 [error] Error: UnboundLocalError: local variable 'start_index' referenced before assignment
    at ae (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2306827)
    at oe (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2304787)
    at Immediate.<anonymous> (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2301081)
    at processImmediate (node:internal/timers:476:21)
2024-07-29 08:49:20.469 [error] [ms-python.python] provider FAILED
2024-07-29 08:49:20.470 [error] Error: UnboundLocalError: local variable 'start_index' referenced before assignment
    at ae (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2306827)
    at oe (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2304787)
    at Immediate.<anonymous> (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2301081)
    at processImmediate (node:internal/timers:476:21)
2024-07-29 08:49:26.120 [error] [ms-python.python] provider FAILED
2024-07-29 08:49:26.120 [error] Error: UnboundLocalError: local variable 'start_index' referenced before assignment
    at ae (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2306827)
    at oe (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2304787)
    at Immediate.<anonymous> (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2301081)
    at processImmediate (node:internal/timers:476:21)
2024-07-29 08:50:55.314 [error] [ms-python.python] provider FAILED
2024-07-29 08:50:55.314 [error] Error: UnboundLocalError: local variable 'start_index' referenced before assignment
    at ae (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2306827)
    at oe (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2304787)
    at Immediate.<anonymous> (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2301081)
    at processImmediate (node:internal/timers:476:21)
2024-07-29 08:51:05.456 [error] [ms-python.python] provider FAILED
2024-07-29 08:51:05.456 [error] Error: UnboundLocalError: local variable 'start_index' referenced before assignment
    at ae (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2306827)
    at oe (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2304787)
    at Immediate.<anonymous> (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2301081)
    at processImmediate (node:internal/timers:476:21)
2024-07-29 08:51:25.051 [error] [ms-python.python] provider FAILED
2024-07-29 08:51:25.052 [error] Error: UnboundLocalError: local variable 'start_index' referenced before assignment
    at ae (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2306827)
    at oe (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2304787)
    at Immediate.<anonymous> (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2301081)
    at processImmediate (node:internal/timers:476:21)
2024-07-29 08:51:30.897 [error] [ms-python.python] provider FAILED
2024-07-29 08:51:30.898 [error] Error: UnboundLocalError: local variable 'start_index' referenced before assignment
    at ae (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2306827)
    at oe (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2304787)
    at Immediate.<anonymous> (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2301081)
    at processImmediate (node:internal/timers:476:21)
2024-07-29 09:36:12.010 [error] [ms-python.python] provider FAILED
2024-07-29 09:36:12.011 [error] Error: UnboundLocalError: local variable 'start_index' referenced before assignment
    at ae (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2306827)
    at oe (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2304787)
    at Immediate.<anonymous> (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2301081)
    at processImmediate (node:internal/timers:476:21)
2024-07-29 09:41:51.518 [error] [ms-python.python] provider FAILED
2024-07-29 09:41:51.518 [error] Error: UnboundLocalError: local variable 'start_index' referenced before assignment
    at ae (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2306827)
    at oe (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2304787)
    at Immediate.<anonymous> (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2301081)
    at processImmediate (node:internal/timers:476:21)
2024-07-29 09:42:46.071 [error] [ms-python.python] provider FAILED
2024-07-29 09:42:46.073 [error] Error: UnboundLocalError: local variable 'start_index' referenced before assignment
    at ae (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2306827)
    at oe (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2304787)
    at Immediate.<anonymous> (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2301081)
    at processImmediate (node:internal/timers:476:21)
2024-07-29 09:44:48.407 [error] [ms-python.python] provider FAILED
2024-07-29 09:44:48.408 [error] Error: UnboundLocalError: local variable 'start_index' referenced before assignment
    at ae (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2306827)
    at oe (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2304787)
    at Immediate.<anonymous> (/google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/out/client/extension.js:2:2301081)
    at processImmediate (node:internal/timers:476:21)
